index.js

```diff
@@ -1,5 +1,7 @@
 const server = require("./data/server.js");
 const PORT = process.env.PORT || 5000;
+const Sentry = require('@sentry/node');
+Sentry.init({ dsn: 'https://c0e164d7a07b4f54854d29fc3c62f50f@sentry.io/5167996' });
```

Comment:
To clarify, it looks like this key is not secret. We can choose to move it to .env but it is not critical. Sentry gives us a public key and a secret key. This is the public key.

https://docs.sentry.io/error-reporting/configuration/?platform=node#dsn